CMakeLists.txt

```diff
@@ -711,7 +712,7 @@ if(NOT SKIP_INSTALL_LIBRARIES AND NOT SKIP_INSTALL_ALL)
 
     install(TARGETS ${MINIZIP_TARGET} ${MINIZIP_DEP}
             EXPORT ${MINIZIP_TARGET}
-            INCLUDES DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/${MINIZIP_TARGET}"
+            INCLUDES DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}"
```

Comment:
I wonder what the historial reason for ${MINIZIP_TARGET} is? Can you check? I'm not feeling well.

Reply:
https://github.com/zlib-ng/minizip-ng/commit/e15e1462f2d6c9b9e9cd63b979223f4f835de40b

Reply:
Perhaps we should remove this change here as it could affect Linux distributions?

Reply:
Say if a target has both minizip and minizip-compat? It would then create two dirs??

Reply:
Maybe we can add multiple INCLUDES DESTINATIONS?